doc/cookbook/source/examples/classifier/LDA.rst

```diff
@@ -0,0 +1,54 @@
+=============================
+Linear Discriminant Analysis
+=============================
+
+LDA learns a linear classifier and requires examples to be CDenseFeatures. The learned linear classification rule is optimal under the assumption that both classes a gaussian distributed with equal co-variance. To find a linear separation :math:`{\bf w}` in training, the in-between class variance is maximized and the within class variance is minimized.
+Three methods to compute the LDA:
+
+Two class Fisher Discriminant Analysis (FLD_LDA):
+
+.. math::
+
+	J({\bf w})=\frac{{\bf w^T} S_B {\bf w}}{{\bf w^T} S_W {\bf w}}
+
+Singular Valued decomposition method (SVD_LDA):  The above derivation of Fisher's LDA requires the invertibility of the within class matrix. However, this condition gets void when there are fewer data-points than dimensions. A solution is to require that :math:`{\bf W}` lies only in the subspace spanned by the data. A basis of the data :math:`{\bf X}` is found using the thin-SVD technique which returns an orthonormal non-square basis matrix :math:`{\bf Q}`. We then require the solution :math:`{\bf w}` to be expressed in this basis.
```

Comment:
Please dont use varialbe names, method names, or enum names in the text, but just let the *code* speak for itself. In the text you can just say that and SVD can be used .. see what I mean? We want to avoid the examples becoming outdated when code changes